exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -68,6 +71,7 @@ func newPRWExporter(cfg *Config, set exporter.CreateSettings) (*prwExporter, err
 		concurrency:     cfg.RemoteWriteQueue.NumConsumers,
 		clientSettings:  &cfg.HTTPClientSettings,
 		settings:        set.TelemetrySettings,
+		retrySettings:   cfg.RetrySettings,
```

Comment:
what are the defaults? are the defaults properly set at this point? I'm asking about the case that `retry_on_failure` is not specified in the yaml configuration for this component.

Reply:
The Default config is taken from here - https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusremotewriteexporter/factory.go#L75-L81